src/ui/public/agg_types/metrics/bucket_avg.js

```diff
@@ -0,0 +1,18 @@
+import AggTypesMetricsMetricAggTypeProvider from 'ui/agg_types/metrics/metric_agg_type';
+import { makeNestedLabel } from './lib/make_nested_label';
+import SiblingPipelineAggHelperProvider from './lib/sibling_pipeline_agg_helper';
+
+export default function AggTypeMetricDerivativeProvider(Private) {
```

Comment:
These names should match how they are being imported.  E.g. `AggTypesMetricsBucketAvgProvider`.  This is why I like named exports, it forces you to use consistent naming. :)  Here and other bucket_* files